src/stages/train.py

```diff
@@ -164,7 +158,8 @@ def train(params: dict) -> None:
         "momentum": BEST_MODEL_PARAMS.get("momentum", 0.5),
         "epochs": EPOCH_SIZE,
         "batch_size_per_worker": GLOBAL_BATCH_SIZE // NUM_WORKERS,
-        "dvc_env": {name: value for name, value in os.environ.items() if name.startswith("DVC")}
+        "dvc_env": {name: value for name, value in os.environ.items() if
+                    name.startswith("DVC") and name != "DVC_STUDIO_TOKEN"}
```

Comment:
Please add this change regardless of what else you keep because it's currently saving your Studio token in a few places.